scripts/ci/check_compliance.py

```diff
@@ -82,9 +82,10 @@ def __init__(self, severity, title, file, line, col=None, desc=""):
         description = f':{desc}' if desc else ''
         msg_body = desc or title
 
-        txt = f'\n{title}{description}\nFile:{file}\nLine:{line}' + \
+        txt = f'\n{title}{description}\nFile:{file}' + \
+              (f'\nLine:{line}' if line else '') + \
```

Comment:
suggestion: `line or ‘’`

Reply:
No, actually that won't work because the string is `\nLine:{line}`